relay-event-schema/src/protocol/contexts/chromium_stability_report.rs

```diff
@@ -0,0 +1,234 @@
+use relay_protocol::{Annotated, Array, Empty, FromValue, IntoValue};
+
+use crate::processor::ProcessValue;
+
+/// Some stability report fields are measured in 4K pages which we convert to bytes.
+fn bytes_from_4k_pages(value: u32) -> u64 {
+    (value as u64) * 4096
+}
+
+/// The state of a process.
+#[derive(Clone, Debug, Default, PartialEq, Empty, FromValue, IntoValue, ProcessValue)]
+pub struct ProcessStateContext {
+    /// The identifier of the process.
+    pub process_id: Annotated<u64>,
+    pub memory_state: Annotated<process_state::MemoryStateContext>,
+    pub file_system_state: Annotated<process_state::FileSystemStateContext>,
+}
+
+/// Nested message and enum types in `ProcessState`.
+pub mod process_state {
```

Comment:
The small nested modules here are a bit unusual, why did you decide to do it like that?